src/MessagingBubble/MessagingBubble.tsx

```diff
@@ -74,15 +76,24 @@ export const MessagingBubble: React.FC<Props> = ({
   return (
     <FlexBox column className={containerClassNames}>
       <FlexBox className={timeAndBubbleContainerClasses}>
-        <span className={timestampClassNames}>{_formatDateForTimestamp(timestamp)}</span>
-        <div className={bubbleClassNames}>
+        {!hideBubble && (
+          <span className={timestampClassNames}>{_formatDateForTimestamp(timestamp)}</span>
+        )}
+        <div className={hideBubble ? null : bubbleClassNames}>
           {replyTo && <div className={replyClassNames}>{replyTo}</div>}
           <Linkify componentDecorator={componentDecorator} matchDecorator={matchDecorator}>
             {children}
           </Linkify>
         </div>
       </FlexBox>
-      {attachments?.length > 0 && <FlexBox className={attachmentClassNames}>{attachments}</FlexBox>}
+      <FlexBox className={timeAndBubbleContainerClasses}>
+        {hideBubble && (
+          <span className={timestampClassNames}>{_formatDateForTimestamp(timestamp)}</span>
+        )}
+        {attachments?.length > 0 && (
+          <FlexBox className={attachmentClassNames}>{attachments}</FlexBox>
+        )}
+      </FlexBox>
```

Comment:
Is there a clean way to set this up so that I can override [these margins](https://github.com/Clever/components/blob/master/src/MessagingAttachment/MessagingAttachment.less#L216-L221)? It's causing there to be extra space between the timestamp and the attachment bubble. Also in quoted announcement bubbles (I think even before this change, on text message too). Also might be easier to just add this as a followup task after bug bash but wanted to highlight here either way

<img width="371" alt="Screen Shot 2021-07-06 at 11 33 56 AM" src="https://user-images.githubusercontent.com/35714960/124689333-0a945d00-de74-11eb-9dde-e7e8409c0501.png">
<img width="330" alt="Screen Shot 2021-07-06 at 11 35 38 AM" src="https://user-images.githubusercontent.com/35714960/124689337-0bc58a00-de74-11eb-9c17-d33ae8ca1c89.png">

Reply:
Is something along these lines not working? 🤔 

```
.MyNewClass {
    .margin--left--none();
}

...

className={message.attachments.length > 0 && message.body === "" && ".MyNewClass"}
```

Reply:
I tried something similar, but the class that's adding the `margin-left: 4rem` is on `MessagingAttachment--Container` which is a descendant within MessagingBubble and QuotedAnnouncementBubble so I think it's later overriding the parent CSS?

Reply:
Actually Cory's recent changes fix the quoted bubble issue and I can try something similar for MessagingBubbles

Reply:
I would've thought we could still do something like this:

```
.MessagingBubble--PathToMessagingAttachmentParentClass .MessagingAttachment--Container {
    .margin--left--none();
}
```

Happy to pair on this later, if helpful! Can also ping @jonah-clever for thoughts on this - they've been in the trenches in Attachments components-land longer than anyone at this point. 😄 

---

Ah, just saw that Cory fixed the quoted bubble version of this! Then yeah, I think a similar approach here might be ideal, if at all possible. 🙌🏻

Reply:
Swear I tried this before 😭 but it works now! https://github.com/Clever/components/pull/681/commits/333e9fd27af65bd7e81b1da8d72f1542f542fa27

<img width="357" alt="Screen Shot 2021-07-07 at 9 10 11 AM" src="https://user-images.githubusercontent.com/35714960/124794559-dc9e2f80-deea-11eb-84ce-2a6ad8537c15.png">